django/template/utils.py

```diff
@@ -106,3 +108,31 @@ def get_app_template_dirs(dirname):
             template_dirs.append(upath(template_dir))
     # Immutable return value because it will be cached and shared by callers.
     return tuple(template_dirs)
+
+
+def property_resolver(arg):
```

Comment:
@timgraham, should I prefix the function with `_` or there is another way to mark a function as private?